### PR TITLE
Add test for Mobi generation

### PR DESCRIPTION
**Pull request by slonopotamus**

### CODE REVIEW

File: spec/cli_spec.rb [hunk 1]
```diff
@@ -15,12 +15,22 @@
...
     infile = example_file 'sample-book.adoc'
     outfile = temp_file 'sample-book.epub'
 
     _, _, res = run_command asciidoctor_epub3_bin, '-a', 'ebook-validate', infile, '-o', outfile
     expect(res.exitstatus).to eq(0)
     expect(File).to exist(outfile)
   end
+
+  # TODO: enable on MacOS when https://github.com/asciidoctor/asciidoctor-epub3/issues/236 is fixed
+  it 'converts sample book to mobi', :skip => !os_darwin? do
```
Comment (mojavelinux) [inline spec/cli_spec.rb]: I tend to just use `unless os_darwin?` after `end`.

If you were to use the hash arguments, it should be:

```
it 'converts sample book to mobi', skip: !os_darwin? do
```

The `:skip => !os_darwin?` is the old Hash syntax.

Comment (slonopotamus) [reply spec/cli_spec.rb]: For some reason, the `skip: !os_darwin?` didn't actually skip it, so I thought I'm passing it the wrong way. 

> I tend to just use `unless os_darwin?` after end.

If I understand how things work properly, this way rspec will have no idea about test existence and will not report it as skipped. Correct?

Comment (slonopotamus) [reply spec/cli_spec.rb]: Okay, I inverted logic accidentally :) I want to skip it _on_ MacOS.

Comment (mojavelinux) [reply spec/cli_spec.rb]: There you go!

**Pull request merged**